proto/dcs.proto

```diff
@@ -8,9 +8,7 @@ message OutTextRequest {
   bool clear_view = 3;
 }
 
-message OutTextResponse {
-  bool success = 1;
-}
+message OutTextResponse {}
```

Comment:
Is returning at least the `success = true` a best practice, or is just returning nothing also fine?

Reply:
Honestly I am not sure. I think nothing is fine since if there was a problem we would be sending an actual error. Based on some reading around of other sources this seems to be fine.